man/tpm2_changeauth.1.md

```diff
@@ -1,53 +1,68 @@
 % tpm2_changeauth(1) tpm2-tools | General Commands Manual
 %
-% SEPTEMBER 2017
+% OCTOBER 2018
 
 # NAME
 
-**tpm2_changeauth**(1) - Insert authorization values for the owner, endorsement
-and lockout authorizations.
+**tpm2_changeauth**(1) - Configuring authorization for TPM OWNER/ENDORSEEMNT/LOCKOUT hierarchies, NV Handles and Transient & Persistent object handles.
 
 # SYNOPSIS
 
 **tpm2_changeauth** [*OPTIONS*]
 
 # DESCRIPTION
 
-**tpm2_changeauth**(1) - set the various (owner, endorse, lockout)
-authorization values.
+**tpm2_changeauth**(1) - Managing authorization passwords for TPM objects/ handles.
+1. Permanent Handles (Owner, Endorsement, Lockout).
+2. Persistent Handles (0x81XX_XXXX).
+3. Transient Handles(0x80XX_XXXX).
+4. NV Handles(0x01XX_XXXX).
 
 # OPTIONS
 
-  * **-o**, **--owner-passwd**=_OWNER\_PASSWORD_:
-    
-    The new owner authorization value. Passwords should follow the "password 
-    authorization formatting standards", see section "Authorization Formatting".
+Passwords should follow the "password authorization formatting standards", see section "Authorization Formatting".
 
-  * **-e**, **--endorse-passwd**=_ENDORSE\_PASSWORD_:
+  * **-o**, **--new-owner-passwd**=_OWNER\_PASSWORD_:
+  * 
+    The new authorization value for the owner hierarchy . 
 
-    The new endorse authorization value. Passwords should follow the same
-    formatting requirements as the **-o** option.
+  * **-e**, **--new-endorsement-passwd**=_ENDORSEMENT\_PASSWORD_:
 
-  * **-l**, **--lockout-passwd**=_LOCKOUT\_PASSWORD_:
+    The new authorization value for the endorsement hierarchy .
 
-    The new lockout authorization value. Passwords should follow the same
-    formatting requirements as the **-o** option.
+  * **-l**, **--new-lockout-passwd**=_LOCKOUT\_PASSWORD_:
 
-  * **-O**, **--old-auth-owner**=_OLD\_OWNER\_AUTH_:
+    The new authorization value for the dictionary lockout.
 
-    The old owner authorization value.
-    Authorization values should follow the password authorization formatting
-    standards, see section "Authorization Formatting".
+  * **-O**, **--current-owner-passwd**=_CURRENT\_OWNER\_AUTH_:
 
-  * **-E**, **--old-auth-endorse**=_OLD\_ENDORSE\_AUTH_:
+    The current authorization value for the owner hierarchy .
 
-    The old endorse authorization value. Authorizations should follow the same
-    formatting requirements as the **-O** option.
+  * **-E**, **--current-endorsement-passwd**=_CURRENT\_ENDORSEMENT\_AUTH_:
 
-  * **-L**, **--old-auth-lockout**=_OLD\_LOCKOUT\_AUTH_:
+    The current authorization value for the endorsement hierarchy.
 
-    The old lockout authorization value. Authorizations should follow the same
-    formatting requirements as the **-O** option.
```

Comment:
why did you drop the formatting requirements? Don't we want to support authorizations like:
-O hmac:foo
-O session:session.ctx

Eventually we see the -S option going away and becoming part of the auth framework.